LICENSE

```diff
@@ -17,3 +17,6 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+All code created or modified by S�bastien Lucas <sebastien@slucas.fr> is licensed
+under GPL 2 (http://www.gnu.org/licenses/gpl.html).
```

Comment:
I don't want to merge this with the license change. I like to keep this library MIT compatible. Can you agree to license your code under MIT?